.travis.yml

```diff
@@ -114,15 +114,15 @@ x-windows-27-shard: &x-windows-27-shard
   <<: *x-windows-shard
   before_install:
   - choco install python2
-  - python -m pip install --upgrade pip
+  - python -m ensurepip --upgrade
   env:
     - *x-tox-env
 
 x-windows-39-shard: &x-windows-39-shard
   <<: *x-windows-shard
   before_install:
-  - choco install python --version 3.9.0
-  - python -m pip install --upgrade pip
+  - choco install python --version 3.9.1
+  - python -m ensurepip --upgrade
```

Comment:
I think this is not the python you think it is. CI says:
```
python2 v2.7.18 [Approved]

python2 package files install completed. Performing other installation steps.

Installing 64-bit python2...

python2 has been installed.

Installed to 'C:\Python27'

Adding C:\Python27 to PATH if needed
...
$ python -m pip install --upgrade pip

The command "python -m pip install --upgrade pip" failed and exited with 1 during .

Your build has been stopped.

C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/opt/bin/python.exe: No module named pip
```

Note chocolatey `C:\Python27` vs `C:/ProgramData/chocolatey/lib/mingw...`. FWICT the latter is the "A basic Python 2.7.9 interpreter is also included: /C/ProgramData/chocolatey/bin/python.exe" referred to at the bottom of this section in the docs: https://docs.travis-ci.com/user/reference/windows/#pre-installed-chocolatey-packages